lib/soapy_cake/admin.rb

```diff
@@ -135,8 +135,8 @@ def verticals
       run Request.new(:admin, :get, :verticals, {})
     end
 
-    def countries
-      run Request.new(:admin, :get, :countries, {})
+    def countries(opts = {})
```

Comment:
Because these entities are currently not persisted in the s3 layer, hence it is not necessary to pass them the opts hash yet. In principle, we should refactor the gem to expose the response parsing method to be directly callable passing the stored XML data. As we currently aim for getting a working prototype of the data pipeline this is currently not in the main scope.